### DISCUSSION TITLE
Database cost on vercel pro hosting plan

### DISCUSSION THREAD

**Discussion by cademis** [vercel/community#4838] [Feature Question] [answered]
### Question

Hi - I plan to purchase the vercel £20 hosting pro plan, is a vercel postgres database included in the price? I see that additional databases are £1 each but I'm wondering about the cost of the single database. Also - for the additional databases - are the usage limits linked? e.g. if I have 2 databases and i use up the limits on one, is the other still usable? Assuming I stay within the limits in this page https://vercel.com/docs/storage/vercel-postgres/usage-and-pricing

**Comment by leerob** [comment, marked as answer]
Yes, it does!
